providers/aws/fargate/region_test.go

```diff
@@ -0,0 +1,30 @@
+package fargate
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// TestAvailableRegions verifies whether Fargate availability in an AWS region is correct.
+func TestAvailableRegions(t *testing.T) {
```

Comment:
I'm not sure what value this test provides. This is equivalent to having two FargateRegions objects with same region strings and then comparing that they are equal.

It also means every time we add a region we have to update this second lists as well. I'd rather not have this test at all.